docs/security/README.md

```diff
@@ -13,6 +13,7 @@ When requesting a security review for your application, please make sure you hav
 * [Secure Coding Practices Quick Reference](https://owasp.org/www-pdf-archive/OWASP_SCP_Quick_Reference_Guide_v2.pdf)
 * [Web Application Security Quick Reference](https://owasp.org/www-pdf-archive//OWASP_Web_Application_Security_Quick_Reference_Guide_0.3.pdf)
 * [Security Mindset/Creating a Security Program Quick Start](https://github.com/OWASP/Quick-Start-Guide/blob/master/OWASP%20Quick%20Start%20Guide.pdf?raw=true)
+* [Non-Functional Requiurements Guidance](../design/design-patterns/non-functional-requirements-investigation-guide.md)
```

Comment:
Same comment as above for observability.

Reply:
Done.